frontend/src/features/archived-items/crawl-list.ts

```diff
@@ -478,10 +268,18 @@ export class CrawlList extends LitElement {
       }
     };
 
-    this.listItems.forEach((el) => {
-      assignProp(el, { name: "role", value: "listitem" });
-      assignProp(el, { name: "collectionId", value: this.collectionId || "" });
-      assignProp(el, { name: "workflowId", value: this.workflowId || "" });
+    this.listItems.forEach((item, i) => {
+      assignProp(item, {
+        name: "collectionId",
+        value: this.collectionId || "",
+      });
+      assignProp(item, { name: "workflowId", value: this.workflowId || "" });
+
+      if (i === 0) {
+        item.classList.remove("border-t");
+      } else {
+        item.classList.add("border-t");
+      }
```

Comment:
Could `first-of-type:` be used here instead of doing this in JS?

Reply:
Yeah, I went with this because `btrix-table-row:first-of-type` won't work with `::slotted`, but it's probably a non-issue to go with `*:first-of-type`.

Reply:
Ahh right, slots. Lemme know if this works with `*:first-of-type`, but otherwise this is fine!